pages/static-security-analysis.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Static Security Analysis
+permalink: /static-security-analysis/
+---
+
+Static analysis is an important part of the development process, and is required for ATO. There are two main types of static security testing that needs to be done: dependency analysis, where the ruby gems, python modules, and js packages your app uses are checked against a list of known vulnerabilities; and code analysis in which your code is checked against a list of antipatterns. There are tools for JS, Ruby, and Python, and you are encouraged to set up this scanning early on in the development cycle to prevent unexpected delays when it's time to get your ATO.
+
+## Javascript
+
+### Dependency analysis
+
+We recommend using the Node Security Project's [NSP tool](https://nodesecurity.io/tools) to scan your package.json or npm-shrinkwrap.json file for known security issues.
+
+To install:
+
+    $ npm install nsp
+
+To run, from your project directory:
+
+    $ nsp check
+
+### Static code analysis
+
+There are several tools available for running analysis on JS projects, but the most full featured seems to be [ESLint](https://eslint.org). ESLint doesn't offer security scanning out of the box, but it is pluggable and Mozilla has provided a set of rules that mimic the behavior of their now deprecated ScanJS. These rules are available on [github](https://github.com/mozfreddyb/eslint-plugin-scanjs-rules).
+
+To install:
+
+    $ npm install eslint
+    $ npm install eslint-plugin-scanjs-rules
+
+___TODO: PROVIDE BASELINE .eslintrc___
+
+To scan, from your project directory:
+
+    $ eslint -c <PATH_TO_ESLINTRC> .
+
+## Ruby
+
+### Dependency analysis
+
+We recommend using [Bundler Audit](https://github/com/rubysec/bundler-audit) to scan your gems for known vulnerabilities. There are other tools available, most notably Gemnasium, but we are currently suggesting Bundler Audit until we can evaluate whether Gemnasium will work as a replacement for all of our languages.
+
+To install:
+
+    $ gem install bundler-audit
+
+To scan, from your project directory:
+
+    $ bundle-audit
+
+### Static code analysis
+
+There are several free and paid services that will do static security analysis of Ruby code, but almost all of them appear to be wrappers around [Brakeman](https://github.com/presidentbeef/brakeman). As such, we suggest going straight to the source.
+
+To install:
+
+    $ gem install brakeman
+
+To scan, from your project directory:
+
+    $ brakeman
+
+## Python
+
+### Dependency analysis
+
+___TODO: DECISION ON GEMNASIUM. IF YES, UPDATE RUBY AND JS SECTIONS ACCORDINGLY___
```

Comment:
I don't think we are ready to _mandate_ it, but Gemnasium does work for python (and ruby, and js), and we have a paid account. No reason not to recommend it.